rsconnect/main.py

```diff
@@ -337,6 +338,13 @@ def deploy_notebook(name, server, api_key, static, new, app_id, title, python, c
 
     click.secho('    Deploying %s to server "%s"' % (file, connect_server.url), fg='white')
 
+    if conda:
+        with cli_feedback('Ensuring conda is supported'):
+            if not is_conda_supported_on_server(connect_server.url):
```

Comment:
```suggestion
            if not is_conda_supported_on_server(connect_server):
```
Needs to be the whole server info, not just the URL.